cloud/storage/core/tools/ci/runner/degradation_test.py

```diff
@@ -0,0 +1,240 @@
+                    lambda x: datetime.datetime.strptime(
+                        x.date, "%Y-%m-%d"
+                    ).date(),
+                    expected_to_grow=True,
+                    name=test_tag,
+                    date=today,
+                    description=f"{suite_kind} {test_tag}",
+                )
+                reports.append(report)
+
+
+def get_monitoring_results(
+    query: str,
+    datapoints: int,
+    from_timestamp: datetime.datetime,
+    to_timestamp: datetime.datetime,
+    profile: str,
+    project_id: str,
+) -> list[list[tp.Tuple[datetime.datetime, float]]]:
+    request = {
+        "container": {
+            "project_id": project_id,
+        },
+        "queries": [{"name": "sensor", "value": query}],
+        "downsampling": {
+            "max_points": datapoints,
+            "grid_aggregation": "GRID_AGGREGATION_AVG",
+            "gap_filling": "GAP_FILLING_PREVIOUS",
+        },
+        "from_time": from_timestamp.strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
+        "to_time": to_timestamp.strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
+    }
+    with tempfile.NamedTemporaryFile() as f:
+        f.write(yaml.dump(request).encode("utf-8"))
+        f.flush()
+
+        result = subprocess.run(
+            [
+                "ycp",
+                "--profile",
+                profile,
+                "monitoring",
+                "v3",
+                "data",
+                "read",
+                "-r",
+                f.file.name,
+            ],
+            capture_output=True,
+        )
+        time.sleep(0.1)
+        if result.returncode != 0:
+            raise RuntimeError(result.stderr)
+        responses = yaml.safe_load(result.stdout)["response_per_query"][0][
+            "timeseries_vector"
+        ]["values"]
+        result = []
+        for response in responses:
+            if "timestamp_values" not in response:
+                continue
+            timestamps = list(
+                map(
+                    lambda x: datetime.datetime.utcfromtimestamp(int(x) / 1000),
+                    response["timestamp_values"]["values"],
+                )
+            )
+            if "double_values" not in response:
+                continue
+            values = response["double_values"]["values"]
+            result.append(list(zip(timestamps, values)))
+        return result
+
+
+def generate_reports_monitoring(
+    profile: str,
+    reports: list[tp.Dict[str, tp.Any]],
+    sensors: list[tp.Dict[str, tp.Any]],
+) -> None:
+    for sensor in sensors:
+        print(sensor)
+
+        results = get_monitoring_results(
+            query=sensor["query"],
+            datapoints=100,
+            from_timestamp=datetime.datetime.now()
+            - datetime.timedelta(days=OVERALL_PERIOD_DAYS),
+            to_timestamp=datetime.datetime.now(),
+            profile=profile,
+            project_id=sensor["project_id"],
+        )
+
+        for result in results:
```

Comment:
There is no way to differentiate between the results in the generated report. Better add something to `description` field which makes it clear.